cfgov/wellbeing/jinja2/wellbeing/about.html

```diff
@@ -25,16 +25,23 @@
 {% endblock %}
 
 {% block content_main %}
+    <h1>{{ _('Why financial well-being?') }}</h1>
 
-    <p>
-      {% if (current_language == 'en') %}
-      <a href="{{ url('fwb_about_es') }}">Ver página en español</a>
-      {% else %}
-      <a href="{{ url('fwb_about_en') }}">View page in English</a>
-      {% endif %}
-    </p>
+    {% set value = {
+        'links': [
+        {
+            'href': url('fwb_about_en'),
+            'language': 'en',
+            'text': 'English'
+        },{
+            'href': url('fwb_about_es'),
+            'language': 'es',
+            'text': 'Spanish'
+        }]
+    } %}
+    {% set temp = value.update({'language':'en'} if (current_language == 'en') else {'language':'es'} )%}
+    {% include "v1/includes/molecules/translation-links.html" %}
```

Comment:
Can we separate out this bit into its own include? Names `fwb-translations.html` or something?

Reply:
I thought of that, but then you still have to pass in the different URLs somehow (wrapping it in a `{% with … %}` block I think?), so it wouldn't be as simple as a one line include (unless we added url sniffing logic to the template itself?). It didn't seem worth the effort, but if you see a clean way I'm happy to update it.

Reply:
Hmmm, I'm not sure I follow. Wouldn't this just work if lines 30-43 were in their own template? Context gets passed automatically yeah?

Reply:
Oh of course! I didn't even notice those were different, even on rechecking ^_^

I'd say this could be done pretty easily with a macro instead of an include, but I don't actually think it's that big of a deal since this code doesn't change often.